pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -2140,52 +2140,71 @@ public PinotResourceManagerResponse dropInstance(String instanceName) {
    * Keeps checking until ideal-state is successfully updated or times out.
    *
    * @param instanceName: Name of Instance for which the status needs to be toggled.
-   * @param toggle: 'True' for ONLINE 'False' for OFFLINE.
-   * @param timeOutInSeconds: Time-out for setting ideal-state.
+   * @param enableInstance: 'True' for enabling the instance and 'False' for disabling the instance.
+   * @param timeOutMs: Time-out for setting ideal-state.
    * @return
    */
-  public PinotResourceManagerResponse toggleInstance(String instanceName, boolean toggle, int timeOutInSeconds) {
+  private PinotResourceManagerResponse enableInstance(String instanceName, boolean enableInstance, long timeOutMs) {
```

Comment:
Not sure why you changed the name of the method to enableInstance. I would keep the method name the same. It takes a boolean that can  be either true or false.

Reply:
By convention, toggle means ENABLE/DISABLE/DROP (Check PinotInstanceRestletResource)

Reply:
In addition, for dropping instance there's already a method called `dropInstance(instanceName)` in `PinotHelixResourceManager`. 
So it's be good to name this method `enableInstance`.

Reply:
I still think the method name should be operation-neutral since it does both. Since 2 out of 3 agree and this is not a very big issue, I will give in.
Some alternative names : "toggleInstance" (like before) or "enableOrDisableInstance()"

Reply:
Actually this is consistent with `HelixAdmin.enableInstance()`